Classes/ToolRegistrator.lua

```diff
@@ -69,17 +69,23 @@ function cToolRegistrator:BindAbsoluteTools()
 		return true
 	end
 
-	local function OnPlayerRightClick(a_Player, a_BlockX, a_BlockY, a_BlockZ, a_BlockFace)
+	local timeSinceLastRightWand = -math.huge
+	local function RightClickWandItem(a_Player, a_BlockX, a_BlockY, a_BlockZ, a_BlockFace)
 		local Succ, Message = GetPlayerState(a_Player).Selection:SetPos(a_BlockX, a_BlockY, a_BlockZ, a_BlockFace, "Second")
 		if (not Succ) then
 			return false
 		end
 
+		if ((os.clock() - timeSinceLastRightWand) < 0.2) then
+			return;
+		end
+
+		timeSinceLastRightWand = os.clock()
```

Comment:
This won't work well with multiplayer, since all players share the same `timeSinceLastRightWand`.

Reply:
This is all inside the `BindAbsoluteTools` function which is called for every player when their playerstate gets created.